plugin/package.json

```
@@ -71,7 +72,7 @@
     "npm-run-all": "^4.1.5",
     "rimraf": "^3.0.2",
     "tmp-promise": "^3.0.3",
-    "typescript": "^4.5.2"
+    "typescript": "4.6.4"
```

upgraded to avoid `ts.resolveTypeReferenceDirective` error after syncing package.json and package-lock.json

---

[dust] Should this be the following instead?

```suggestion
    "typescript": "^4.6.4"
```

Or do we want to start pinning to specific versions to prevent issues going forward during package upgrades?